server/StudySharp.API/Requests/TheoryBlocks/GetTheoryBlockByIdRequest.cs

```diff
@@ -0,0 +1,7 @@
+namespace StudySharp.API.Requests.TheoryBlocks
+{
+    public class GetTheoryBlockByIdRequest
+    {
+        public int Id { get; set; }
```

Comment:
![image](https://user-images.githubusercontent.com/45018119/134784991-ab7a4a92-f768-4296-8f99-889d943295f8.png)

Reply:
Do the same but to GetTheoryBlockByIdRequest accordingly